mlflow/gateway/schemas/completions.py

```diff
@@ -30,9 +30,9 @@ class Candidate(ResponseModel):
 
 
 class Metadata(ResponseModel):
-    input_tokens: Optional[int]
-    output_tokens: Optional[int]
-    total_tokens: Optional[int]
```

Comment:
I saw some other optional fields without default value, like
```
class CandidateMetadata(ResponseModel, extra=Extra.allow):
    finish_reason: Optional[FinishReason]
```

Reply:
They should be fixed as well